cronjobs/initiate_marc_pipeline.py

```diff
@@ -38,11 +41,81 @@ def OptimizeSolrIndex(index):
         sys.exit(-1)
 
 
+def GetPPNsInIndex(index):
+   try:
+      url = "http://localhost:8983/solr/biblio/export"
```

Comment:
Shouldn't the index variable be used to build the url instead of hardcoding biblio?